rules/falco_rules.yaml

```diff
@@ -2575,22 +2575,26 @@
       evt.arg.name endswith "zsh_history" or
       evt.arg.name contains "fish_read_history" or
       evt.arg.name endswith "fish_history" or
+      evt.arg.name contains "ash_history" or
       evt.arg.oldpath contains "bash_history" or
       evt.arg.oldpath endswith "zsh_history" or
       evt.arg.oldpath contains "fish_read_history" or
       evt.arg.oldpath endswith "fish_history" or
+      evt.arg.oldpath contains "ash_history" or
```

Comment:
Would be possible to use endswith operator instead of contains?
Thanks

Reply:
Good idea @darryk10 :+1:

Reply:
If we introduce `ash_history` (doesn't matter whether using operator `contains` or `endswith`), we can ignore the `bash_history` then. A proper comment will be useful here saying both `bash_history` and `ash_history` will be covered.

Reply:
Seems like a reasonable approach to me. Do we still want to use `contains` or do we want to change it to `endswith`. I don't have a preference.